etc/afpd/dircache.c

```diff
@@ -123,6 +123,12 @@ static unsigned int dircache_metadata_window = DEFAULT_DIRCACHE_METADATA_WINDOW;
 static unsigned int dircache_metadata_threshold =
     DEFAULT_DIRCACHE_METADATA_THRESHOLD;
 
+/*
+ * Configuration variable to control whether files are allowed in the dircache.
+ * When false (default), files are rejected.
+ */
+bool dircache_files_enabled = false;  // Default: false (reject files)
```

Comment:
nitpick: we want to use C style code comments consistently!

and besides, the comment seems a bit redundant, it's obvious from the context what this does (to my eyes)